### DISCUSSION TITLE
AC signals - what do I need to do to protect my PC and Amp?

### DISCUSSION THREAD

**Discussion by Ho-Ro** [OpenHantek/OpenHantek6022#310] [Q&A] [answered]
@bedubyasa asked:
> I have purchased a Hantek 6022BE scope with the aim of troubleshooting my Harmaon Kardon Integrated amplifier. I have read of problems looking at AC signals - what do I need to do to protect my PC and Amp?

**Comment by Ho-Ro** [comment, marked as answer]
I've moved this question to Q&A because it fits better here.

Please note that via USB there is a direct (low impedance) ground connection from the BNC sockets to the PC and thus to the protective earth. Therefore, it is a totally bad idea to connect the ground terminal to an unknown potential, because then high currents can flow that can damage your scope or your PC. This is typical for most USB scopes, not only the 
Hantek. Dave has good information on this topic:
https://youtu.be/xaELqAo4kkQ

Another point is that the Hantek only has DC coupling, but you can either use a DC breaker or simply connect a capacitor (100nF/250V) in front of the probe. If you want the de-luxe solution, check out my AC modification, it's supported by OH6022:
https://github.com/OpenHantek/OpenHantek6022/blob/main/docs/HANTEK6022_AC_Modification.pdf

**Reply by bedubyasa** [reply]
Many thanks for your answer!
